library/spdm_requester_lib/libspdm_req_get_event_types.c

```diff
@@ -125,6 +125,11 @@ static libspdm_return_t libspdm_try_get_event_types(libspdm_context_t *spdm_cont
         status = LIBSPDM_STATUS_BUFFER_TOO_SMALL;
         goto receive_done;
     }
+    if (spdm_response_size != sizeof(spdm_supported_event_types_response_t) +
```

Comment:
I think we still need `if (spdm_response_size < sizeof(spdm_supported_event_types_response_t)) {`  before line 121.
It is to prevent invalid read access for `spdm_response->supported_event_groups_list_len`.

Reply:
Agree.